Adyen/UI/Form/FormViewController.swift

```diff
@@ -11,42 +11,75 @@ import UIKit
 @_spi(AdyenInternal)
 open class FormViewController: UIViewController, AdyenObserver, PreferredContentSizeConsumer {
 
-    fileprivate enum Animations {
-        fileprivate static let keyboardBottomInset = "keyboardBottomInset"
-        fileprivate static let firstResponder = "firstResponder"
+    private enum Animations {
+        static let keyboardBottomInset = "keyboardBottomInset"
+        static let firstResponder = "firstResponder"
     }
 
+    // MARK: - UI elements
+
+    private let scrollView: UIScrollView = {
```

Comment:
this should be a lazy var so it only gets instantiated if it's actually needed

Reply:
I was thinking maybe we could use a composition instead. Like create a `FormScrollViewController` that has embedded `FormViewController` (inside the `UIScrollView`) so this class would be responsible **ONLY** for scroll view management and not mess up the existing code. But so far my code knowledge is limited so I'm just throwing a possible option.
So then we can define the view controller behavior upon it's instantiation choosing between `FormViewController` and `FormScrollViewController` thus keeping all components not coupled and not knowing anything about each other.
Extracting this to an assembly layer would also make it much easier to produce whatever kind we need in unit tests.

Reply:
Let's have a quick whiteboarding session later on. My idea doesn't lead to any code duplication, at least in my mindmap! :)

Reply:
As discussed in a previous mob session, we will build the case to refactor internally how the drop-in is rendered. In a way that no scrolling should be present in the `FormViewController`.